src/PIL/ImageFont.py

```diff
@@ -927,7 +932,10 @@ def load_path(filename: str | bytes) -> ImageFont:
             return load(os.path.join(directory, filename))
         except OSError:
             pass
-    msg = "cannot find font file"
+    msg = f"cannot find font file '{filename}' in `sys.path`"
+    if os.path.exists(filename):
+        msg += f', did you mean `ImageFont.load("{filename}")` instead?'
```

Comment:
Looks good, thanks!

I think we can skip these backticks, it's understandable without them:

```suggestion
    msg = f"cannot find font file '{filename}' in sys.path"
    if os.path.exists(filename):
        msg += f', did you mean ImageFont.load("{filename}") instead?'
```